src/DIPS.Xamarin.UI.Android/DIPS.Xamarin.UI.Android.csproj

```diff
@@ -55,17 +57,20 @@
     <Compile Include="Resources\Resource.designer.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="InternalButtonRenderer.cs" />
+    <Compile Include="VibrationService.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="Resources\AboutResources.txt" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Xamarin.AndroidX.MediaRouter" Version="1.2.0" />

```

Comment:
Do these have to be included in the nuspec? @haavamoa

Reply:
If its not a transitive dependency, then yes.